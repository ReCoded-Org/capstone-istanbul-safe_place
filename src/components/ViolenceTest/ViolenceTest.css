
.violence-intro{
    background-color: #C8D7C2;
    padding:5rem 0;
}

.violence-test {
    display:flex;
    justify-content: space-between;
    align-items: baseline;
}

h2 {
    color: #FE8F7C;
    padding-top:1rem;
}
p, h5{
    color: #FFFEFC;
    text-align: justify;
}

span {
    font-size: .9rem;
}
Button {
    color:#FFFEFC !important;
    background-color: #FE8F7C !important;
    border: none !important;
}


/* 


 .violence-intro{
    background-color: #C8D7C2;
    padding :12rem 15rem;
    display: flex;
    justify-content: space-evenly;
    flex-wrap: wrap;
}
.violence-img{
 width:45%;
 height: auto;
}
.second-section{
 padding-left: 2rem;
  width:45%;
  height: auto;
  display: flex;
  flex-direction: column;
 justify-content: space-between;
}

.violence-images{
    display: flex;
    flex-wrap: wrap;
   justify-content: space-between;
}


.violence-test div {
    display: flex;
    justify-content: space-between;
} */
