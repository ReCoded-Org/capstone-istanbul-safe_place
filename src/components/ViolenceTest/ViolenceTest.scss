
$warningColor: #ffbfb5;
$darkColor: #4f4f4f;
$lightColor: #fffefc;
@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
$mainFontFamily: "Roboto", sans-serif;

.violence-intro{
    background-color: #C8D7C2;
    font-family: Roboto;
    padding:5rem 0;

    .violence-test {
        display:flex;
        justify-content: space-between;
        align-items: flex-end;
        align-content:flex-end;
        flex-wrap: wrap;
    }
    
    h3 {
        color: #FE8F7C;
        padding-top:.8rem;
    }
    p, h5{
        color: #FFFEFC;
        text-align: justify;
    }
    
     span {
        font-size: .9rem;
    }
    Button {
        color:#FFFEFC !important;
        background-color: #FE8F7C !important;
        border: none !important;
    }
    img{
      width:100%;
      margin-top:1rem;
    }
}

