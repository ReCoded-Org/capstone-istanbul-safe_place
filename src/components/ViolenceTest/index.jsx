import React from "react";
import { Container, Col, Row, Button } from "react-bootstrap";
import { Link } from "react-router-dom";
import trappedWomen from "../../images/trappedWomen.png";
import dontSpeak from "../../images/dontSpeak.png";
import womenStopViolence from "../../images/womenStopViolence.png";
import "./index.scss";

const ViolenceTest = () => {
  return (
    <div className="violenceIntro">
      <Container>
        <Row className="justify-content-md-center">
          <Col lg={6} md={12}>
            <img src={trappedWomen} alt="A woman trapped by fear" />
          </Col>
          <Col lg={6} md={12}>
            <Row>
              <Col>
                <img src={dontSpeak} alt="A man not allowing woman to speak" />
              </Col>
              <Col>
                <img
                  src={womenStopViolence}
                  alt="A women saying stop to violence"
                />
              </Col>
            </Row>
            <Row>
              <Col>
                <h3>What is violence against women?</h3>
                <p>
                  According to the first article of the Declaration on the
                  Prevention of Violence Against Women, adopted by the United
                  Nations General Assembly in 1993; It is the threat or coercion
                  of any kind of gender-based violence or act that results in or
                  is likely to result in physical, sexual and emotional harm to
                  women in the public and private spheres, and arbitrarily
                  restricting freedom.
                </p>
                <div className="violenceTest">
                  <h5>
                    Are you in an abusive relationship?
                    <span>Take our test</span>
                  </h5>
                  <Link to="/test">
                    <Button> Find out now</Button>
                  </Link>
                </div>
              </Col>
            </Row>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default ViolenceTest;
