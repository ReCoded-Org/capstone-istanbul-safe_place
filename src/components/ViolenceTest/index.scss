@import "../../style/variables";

/* Colors */

.violenceIntro {
  background-color: $secondaryColor;
  font-family: Roboto;
  padding: 5rem 0;

  .violenceTest {
    display: flex;
    justify-content: space-between;
    align-items: flex-end;
    align-content: flex-end;
    flex-wrap: wrap;
  }

  h3 {
    color: $primaryColor;
    padding-top: 0.8rem;
  }

  p,
  h5 {
    color: #fffefc;
    text-align: justify;
  }

  span {
    font-size: 0.9rem;
  }

  button {
    color: $lightColor;
    background-color: $primaryColor;
    border: none;

    &:hover {
      background-color: $lightColor;
      color: $primaryColor;
    }
  }

  img {
    width: 100%;
    margin-top: 1rem;
  }
}
