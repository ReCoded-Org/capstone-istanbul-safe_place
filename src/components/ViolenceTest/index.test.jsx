import React from "react";
import renderer from "react-test-renderer";
import ViolenceTest from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<ViolenceTest />).toJSON();
  expect(tree).toMatchSnapshot();
});

