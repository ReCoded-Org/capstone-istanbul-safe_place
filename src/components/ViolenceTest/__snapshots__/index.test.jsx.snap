// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="violence-intro"
>
  <div
    className="container"
  >
    <div
      className="justify-content-md-center row"
    >
      <div
        className="col-lg-6 col-md-12"
      >
        <img
          alt="img1"
          className="img1"
          src="violence-test-image1.png"
        />
      </div>
      <div
        className="col-lg-6 col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <img
              alt="img2"
              src="violence-test-image2.png"
            />
          </div>
          <div
            className="col"
          >
            <img
              alt="img3"
              src="violence-test-image3.png"
            />
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            <h3>
              What is violance against women?
            </h3>
            <p>
              According to the first article of the Declaration on the Prevention of Violence Against Women, adopted by the United Nations General Assembly in 1993; It is the threat or coercion of any kind of gender-based violence or act that results in or is likely to result in physical, sexual and emotional harm to women in the public and private spheres, and arbitrarily restricting freedom.
            </p>
            <div
              className="violence-test"
            >
              <h5>
                Are you in an abusive relationship?
                <span>
                  Take our test
                </span>
              </h5>
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                 Find out now
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
